src/main/Config.cpp

```diff
@@ -1082,16 +1081,20 @@ Config::processConfig(std::shared_ptr<cpptoml::table> t)
                          "is ignored. Please remove from config");
                  }},
                 {"DEPRECATED_SQL_LEDGER_STATE",
-                 [&]() { DEPRECATED_SQL_LEDGER_STATE = readBool(item); }},
+                 [&]() {
+                     CLOG_WARNING(
```

Comment:
Are we ok with silently migrating to BucketListDB? (and deleting ledger state tables without explicitly flagging this to operators)

Reply:
Yeah, we've given as much warning as we can, and the initial `DEPRECATED_SQL_LEDGER_STATE` was intended to be the warning flag. It doesn't matter at this point if state tables get dropped since the node would fall of the network soon anyway.